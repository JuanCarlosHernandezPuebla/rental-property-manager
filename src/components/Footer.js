import React, { Component } from 'react';

class Footer extends Component {

    render() {
        return(
            <footer>
                <div className="container text-center">

                    <div className="row text-center">

                        <div className="col-md-3">
                            <h3>Features</h3>
                            <p>A paragraph</p>
                        </div>

                        <div className="col-md-3">
                            <h3>Resources</h3>
                        </div>

                        <div className="col-md-3">
                            <h3>Create Account</h3>
                        </div>

                    </div>
                </div>
            </footer>
        );
    }
    
}

export default Footer;