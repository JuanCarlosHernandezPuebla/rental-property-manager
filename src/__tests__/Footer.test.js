import React from 'react';
import Footer from '../components/Footer';
import renderer from 'react-test-renderer';

test('Footer render test', () => {
    const component = renderer.create(
        <Footer/>
    );
    expect(component.toJSON()).toMatchSnapshot();
});