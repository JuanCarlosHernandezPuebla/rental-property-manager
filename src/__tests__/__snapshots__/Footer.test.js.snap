// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render test 1`] = `
<footer>
  <div
    className="container text-center"
  >
    <div
      className="row text-center"
    >
      <div
        className="col-md-3"
      >
        <h3>
          Features
        </h3>
        <p>
          A paragraph
        </p>
      </div>
      <div
        className="col-md-3"
      >
        <h3>
          Resources
        </h3>
      </div>
      <div
        className="col-md-3"
      >
        <h3>
          Create Account
        </h3>
      </div>
    </div>
  </div>
</footer>
`;
